index.html

```diff
@@ -116,50 +100,100 @@ <h5 class="card-title text-center">Cicerone</h5>
 
-                <img src="assets/home/web.png" class="abs">
-                <div class="web">
-                    <p class=" text-center">Web Development</p>
+                <img src="assets/home/human.png" class="abs">
+                <div class="adv">
+                    <br>
+                    <p class="text-center">Advanced Artificial Intelligence</p>
                 </div>
             </div>
             <div class="col-lg-6 col-md-6 col-sm-12  text-center rel">
 
                 <img src="assets/home/cyber.png" class="abs">
                 <div class="cyber">
-
+                    <br>
                     <p class=" text-center">Cyber Security</p>
                 </div>
             </div>
+            <div class="col-lg-6 col-md-6 col-sm-12 text-center rel">
+
+                <img src="assets/home/ml.png" class="abs">
+                <div class="app">
+                    <br>
+                    <p class=" text-center">App Development</p>
+                </div>
+            </div>
             <div class="col-lg-6 col-md-6 col-sm-12 text-center rel">
 
                 <img src="assets/home/human.png" class="abs">
-                <div class="human">
+                <div class="sci">
+                    <br>
+                    <p class=" text-center">Theoretical Computer Science</p>
+                </div>
+            </div>
+            <div class="col-lg-6 col-md-6 col-sm-12 rel text-center">
 
-                    <p class=" text-center">Human Computer Interaction</p>
+                <img src="assets/home/web.png" class="abs">
+                <div class="web">
+                    <br>
+                    <p class="text-center">Web Development</p>
                 </div>
             </div>
-            <div class="col-lg-6 col-md-6 col-sm-12 text-center rel">
+            <div class="col-lg-6 col-md-6 col-sm-12 rel text-center">
+
+                <img src="assets/home/cyber.png" class="abs">
+                <div class="block">
+                    <br>
+                    <p class="text-center">Blockchain</p>
+                </div>
+            </div>
+            <div class="col-lg-6 col-md-6 col-sm-12 rel text-center">
 
                 <img src="assets/home/ml.png" class="abs">
-                <div class="ml">
+                <div class="quant">
+                    <br>
+                    <p class="text-center">Quantum Computing</p>
+                </div>
+            </div>
+            <div class="col-lg-6 col-md-6 col-sm-12 rel text-center">
 
-                    <p class=" text-center">Machine Learning</p>
+                <img src="assets/home/web.png" class="abs">
+                <div class="creat">
+                    <br>
+                    <p class="text-center">Creatives</p>
                 </div>
             </div>
-        </div>
+            <div class="col-lg-6 col-md-6 col-sm-12 rel text-center">
+
+                <img src="assets/home/cyber.png" class="abs">
```

Comment:
Prefer using relative path in `src`

Reply:
How do I provide relative path for this URL?